addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2557,11 +2557,21 @@ export class OdooEditor extends EventTarget {
      * @private
      */
     _onSelectionChange() {
+        const selection = this.document.getSelection();
+        // When CTRL+A in the editor, sometimes the browser use the
+        // editable element as an anchor & focus node
+        // this is an issue for the commands and the toolbar.
+        // So we need to "fix" the selection to be base on the editable children
```

Comment:
Multiple issues here:
* `base` => `based`
* Please wrap the comment correctly at 80 chars.
* Capitalization and punctiation are weird. I suppose the first sentence (ending on second line)  should have a dot, so that the sentence starting with "this" would be capitalized and not end with a dot so that the sentence starting with "So" would then be lowercase as it is the continuation as the previous sentence. Then it can end with a final dot.
* I'd remove the quotes around "fix". Actually that makes me think... shouldn't we use `getDeepRange` in this case ? See comment below.

Reply:
:ok_hand: